includes/omise-wc-setting.php

```diff
@@ -0,0 +1,100 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+return array(
```

Comment:
@nimid from the commit that you removed text domain out from the `__()` method, https://github.com/omise/omise-woocommerce/pull/23/commits/4e6b1ea3bcedcbec6ddc8931afdf7a5adfe007f6

I saw article that mentions about text domain here https://codex.wordpress.org/I18n_for_WordPress_Developers#Text_Domains

Would be better to not remove `text domain` here, but, change to some variable instead?
something like,
```php
$text_domain = 'Omise';

__( 'Enable/Disable', $text_domain );
```
(at least for this time until we merge #24 )?